src/main/java/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,26 @@
+package service.impl;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import model.FruitTransaction;
+import service.DataConverter;
+
+public class DataConverterImpl implements DataConverter {
+    private static final int ZERO_ELEMENT = 0;
+    private static final int ONE_ELEMENT = 1;
+    private static final int TWO_ELEMENT = 2;
```

Comment:
It's recommended to use more descriptive names for your constants. The names `ZERO_ELEMENT`, `ONE_ELEMENT` and `TWO_ELEMENT` don't give a clear understanding of what these constants represent. Consider renaming them to something like `OPERATION_TYPE_INDEX`, `FRUIT_TYPE_INDEX` and `QUANTITY_INDEX`.